libbpf-cargo/src/btf/c_types.rs

```diff
@@ -32,6 +32,14 @@ pub struct btf_enum {
     pub val: i32,
 }
 
+#[repr(C)]
+#[derive(Debug, Clone, DerivePread, Pwrite, IOread, SizeWith)]
+pub struct btf_enum64 {
```

Comment:
Can we use the [`libbpf_sys` definition](https://docs.rs/libbpf-sys/1.1.1+v1.1.0/libbpf_sys/struct.btf_enum64.html) instead?

Reply:
libbpf_sys::btf_enum64 does NOT derive `DerivePread`.

Reply:
I missed that. That's rather unfortunate :-|